app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,117 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui padded segment">
+      <div class="ui yellow inverted segment">
+        <div class="ui tablet stackable grid">
+          <div class="three wide column" style="font-size:24px;margin:10px 0 0 0">
+            08:48
+          </div>
+          <div class="thirteen wide column">
+            <p>
+              {{t 'Please complete registration within 10:00 minutes.'}}
+              <br>
+              {{t 'After 10:00 minutes, the reservation we\'re holding will be released to others.'}}
+            </p>
+          </div>
+        </div>
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Buyer'}}
+      </h4>
+      <div class="field">
+        <label class="required" for="firstname">{{t 'First Name'}}</label>
+        <input type="text" name="firstname">
+      </div>
+      <div class="field">
+        <label class="required" for="lastname">{{t 'Last Name'}}</label>
+        <input type="text" name="lastname">
+      </div>
+      <div class="field">
+        <label class="required" for="email">{{t 'Email'}}</label>
+        <input type="text" name="email">
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Holder\'s Information'}}
+      </h4>
+      <div class="field">
+        <label class="required" for="first_name">{{t 'First Name'}}</label>
+        <input type="text" name="first_name">
+      </div>
+      <div class="field">
+        <label class="required" for="last_name">{{t 'Last Name'}}</label>
+        <input type="text" name="last_name">
+      </div>
+      <div class="field">
+        <label class="required" for="_email">{{t 'Email'}}</label>
+        <input type="text" name="_email">
+      </div>
+      <div class="field">
+        <label>{{t 'Autofill'}}</label>
+        <input type="checkbox" tabindex="0" class="hidden">
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Billing Information'}}
+      </h4>
+      <div class="field">
+        <label class="required">{{t 'Select country'}}</label>
+        {{#ui-dropdown class='search selection'}}
+          {{input type='hidden' id='country'}}
+          <i class="dropdown icon"></i>
+          <div class="default text">{{t 'Select country'}}</div>
+          <div class="menu">
+            <div class="item" data-value="af"><i class="af flag"></i>Afghanistan</div>
```

Comment:
@harshitagupta30 please use the list of countries from `utils/dictionary/demography.js`